challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map { |animal| animal.capitalize  }
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  input.detect {|animal| animal == "dog"}.split
```

Comment:
Does this return `["d", "o", "g"]`? I think we're expecting `["dog"]` here.

Reply:
Oops. My guess was wrong. I checked this `irb`:

```
irb> "dog".split
#=> ["dog"]
```

Anywho ... there's another method instead of `detect` that would return `["dog"]` directly. That usage of `split` surprises me.